packages/react/src/lib/accessibility/FocusZone/AutoFocusZone.tsx

```diff
@@ -0,0 +1,245 @@
+import * as React from 'react'
+import * as ReactDOM from 'react-dom'
+import * as keyboardKey from 'keyboard-key'
+import * as PropTypes from 'prop-types'
+import * as _ from 'lodash'
+import { EventStack } from '../..'
+
+import {
+  getNextElement,
+  getFirstTabbable,
+  getLastTabbable,
+  getWindow,
+  focusAsync,
+} from './focusUtilities'
+
+import { AutoFocusZoneProps } from './AutoFocusZone.types'
+import getUnhandledProps from '../../getUnhandledProps'
+import getElementType from '../../getElementType'
+import * as customPropTypes from '../../customPropTypes'
+
+/** AutoFocusZone is used to focus the first element inside it's children */
+export class AutoFocusZone extends React.Component<AutoFocusZoneProps, {}> {
```

Comment:
`AutoFocusZone` has common logic with `FocusTrapZone` - focusing inner element on componentDidMount(). `FocusTrapZone` can extend `AutoFocusZone`'s and reuse that logic. So basically we'll get
```
<FocusTrapZone>
   <AutoFocusZone>
   </AutoFocusZone>
</FocusTrapZone>
```

Reply:
Can you check the latest changes? `FocusTrapZone`, now returns in the render method `AutoFocusZone`. Is that what you mean with this comment?

Reply:
yes 👍